### PR TITLE
452 upgrade views

### PR DESCRIPTION
**Pull request by acouch**
### Acceptance Test

- [x] Passes all tests

### CODE REVIEW

**Comment by acouch** [comment]
QA site created: http://452_upgrade_views.dkan.qa.nuamsdev.com

File: build-dkan.make [hunk 1]
```diff
@@ -8,4 +8,4 @@ includes[core] = drupal-org-core.make
 projects[dkan][type] = profile
 projects[dkan][download][type] = git
 projects[dkan][download][url] = https://github.com/NuCivic/dkan.git
-projects[dkan][download][branch] = 7.x-1.x
+projects[dkan][download][branch] = 452_upgrade_views
```
Comment (drkloc) [inline build-dkan.make]: travis use this?

Comment (drkloc) [inline build-dkan.make]: nevermind, the qa site must use this.

**Comment by acouch** [comment]
ref #453 

This is ready to pull.

**Comment by drkloc** [comment]
this is going to get in with NuCIvic/dkan_dataset#113. Closing this.

**Comment by acouch** [comment]
QA site created: http://452_upgrade_views.dkan.qa.nuamsdev.com

**Pull request closed without merging**